rush.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.62.1",
-  "pnpmVersion": "6.22.1",
+  "rushVersion": "5.121.0",
+  "pnpmVersion": "8.15.7",
```

Comment:
pnpm@9 is technically "latest", but it was just released last week so I'd rather wait a few months for it to bake.  pnpm@8 will be maintained for years to come.